bin/plugins/scanner/__init__.py

```diff
@@ -107,7 +115,7 @@ def scan(datas, wait_server=False):
                                 {
                                     'name': document_name,
                                     'description': attachment['notes'],
-                                    'category_id': attachment['category'][:2],
+                                    'category_id': category_id,
```

Comment:
before it was a string! :fearful: let's make sure that integer format works

Reply:
I've tested it and works :+1: